scipy/sparse/linalg/dsolve/SuperLU/SRC/msc_stdint.h

```diff
@@ -33,6 +33,11 @@
 #error "Use this header only with Microsoft Visual C++ compilers!"
 #endif // _MSC_VER ]
 
+#if _MSC_VER >= 1900
```

Comment:
It's cleaner when doing diffs and the like. How many places would need to be changed under @pv's alternative solution?